src/XRP/serializer.ts

```diff
@@ -77,6 +78,7 @@ interface PathElementJSON {
   currencyCode?: string
 }
 
+type UnauthorizeJSON = string
```

Comment:
will this eventually be an object-y type? although i suppose that a string is valid JSON